src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -6461,4 +6461,32 @@ inline void compute_remap_src_loc(Rpp32f rowLoc, Rpp32f colLoc, Rpp32s &srcLoc,
     srcLoc = (rowLoc * stride) + colLoc * channels;
 }
 
+inline void rpp_pixel_check_and_store(float pixel, Rpp8u* dst)
+{
+    pixel = fmax(fminf(pixel, 255), 0);
```

Comment:
It is the same
This is a overloaded function and called inside a helper function inside box filter kernel
https://github.com/sampath1117/rpp/blob/sr/box_filter_host/src/modules/cpu/kernel/box_filter.hpp#L80

Added this so that corresponding function will be called for corresponding datatype